.github/workflows/release.yml

```diff
@@ -135,7 +135,21 @@ jobs:
       - name: Push images with version tag
         env:
           CTR_TAG: ${{ needs.version.outputs.version }}
-        run: make docker-push
+          IMAGE_REPO: openservicemesh
+        run: |
+          for image_tag in bookbuyer bookthief bookstore bookwarehouse tcp-echo-server tcp-client init osm-controller
+            do
+              tokenUri="https://auth.docker.io/token?service=registry.docker.io&scope=repository:${IMAGE_REPO}/$image_tag:pull"
+              bearerToken="$(curl --silent --get $tokenUri | jq --raw-output '.token')"
+              listUri="https://registry-1.docker.io/v2/${IMAGE_REPO}/$image_tag/tags/list"
+              authz="Authorization: Bearer $bearerToken"
+              version_list="$(curl --silent --get -H "Accept: application/json" -H "$authz" $listUri | jq --raw-output '.')"
+              exists=$(echo $version_list | jq --arg t ${CTR_TAG} '.tags | index($t)')
+              if [[ $exists == null ]]
+              then
+                make docker-push-$image_tag
+              fi
+            done
```

Comment:
Can we run this via a command (ex. `make publish-images`) and set the required env?
/cc @nojnhuh

Reply:
I don't see any issue with making this available via the Makefile or other script.